src/main/java/com/sublinks/sublinksapi/person/entities/Person.java

```diff
@@ -65,11 +65,7 @@ public class Person implements UserDetails, Principal {
   Set<LinkPersonPost> linkPersonPost;
 
   @ManyToOne
-  @JoinTable(
-      name = "link_person_instances",
-      joinColumns = @JoinColumn(name = "person_id"),
-      inverseJoinColumns = @JoinColumn(name = "instance_id")
-  )
+  @JoinTable(name = "link_person_instances", joinColumns = @JoinColumn(name = "person_id"), inverseJoinColumns = @JoinColumn(name = "instance_id"))
```

Comment:
@jgrim the editorconfig must be broken, because I can't see how this is better than the previous change

Reply:
The editorconfig was wrong.